### PR TITLE
Fix che-in-che devfile to use reference

### PR DESCRIPTION
**Pull request by mshaposhnik**
### What does this PR do?
Fix che-in-che devfile to use reference field  due to https://github.com/eclipse/che/issues/12733

### CODE REVIEW

**Review by skabashnyuk — APPROVED** [review]

File: devfile.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ components:
     id: che-machine-exec-plugin:0.0.1
   - name: che-dev
     type: kubernetes
-    local: .che-dev.yaml
+    rerefence: .che-dev.yaml
```
Comment (sleshchenko) [inline devfile.yaml]: It is not related to this PR, just a question: Should it be still hidden even if devfile is not? I mean should not this file be renamed to `che-dev.yaml`

**Review by sleshchenko — APPROVED** [review]
:rocket:

File: devfile.yaml (hunk 1 above)
Comment (mshaposhnik) [reply devfile.yaml]: I dunno. Author is next seat and we can ask him :)

**Comment by mshaposhnik** [comment]
ci-build

**Comment by mshaposhnik** [comment]
ci-build

**Pull request merged**